Description: use debian library version of liblz4 rather than the embedded copy
 Currently this is almost a no-op since lz4 support is disabled.
Author: Anthony F McInerney <afm404@gmail.com>, Andreas Metzler <ametzler@debian.org>
Last-Update: 2018-05-01

--- a/src/bin/termptysave.c
+++ b/src/bin/termptysave.c
@@ -2,7 +2,7 @@
 #include <Elementary.h>
 #include "termpty.h"
 #include "termptysave.h"
-#include "lz4/lz4.h"
+#include <lz4.h>
 #include <sys/mman.h>
 
 #if defined (__MacOSX__) || (defined (__MACH__) && defined (__APPLE__))
--- a/src/bin/meson.build
+++ b/src/bin/meson.build
@@ -31,7 +31,6 @@ terminology_sources = ['private.h', 'cov
                        'termptygfx.c', 'termptygfx.h',
                        'termptyext.c', 'termptyext.h',
                        'termptysave.c', 'termptysave.h',
-                       'lz4/lz4.c', 'lz4/lz4.h',
                        'md5/md5.c', 'md5/md5.h',
                        'utf8.c', 'utf8.h',
                        'win.c', 'win.h',
